yetanotherco/aligned_layer
# feat (risc0) add public input as separate parameter on verification data

- Cli reads pub input for risc0
- Batcher checks pub input matches journal on risc0
- Operator checks pub input matches journal on risc0
- Fibonacci example saves pub input to file

# To test
1. Start everything as normal
2. Run ```make batcher_send_risc0_burst``` and check its verified normal
